script.js

```diff
@@ -1,11 +1,22 @@
+
 // Create an array called `colors` that contains three colors.
+var colors = ['red', 'green', 'blue'];
 
 // Log the last item in the array to the console.
-
+colors[2];
+console.log("blue");
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+var colors();
+for ( i in colors) {
+  console.log("My favorite color is", colors[2]);
+}
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  name: "John",
+  gender: "male",
+  school: "highschool"
+}
 // Add a `facial-hair` property to `instructor`.
-
+instructor.'facial-hair' = "mustache"
```

Comment:
Because it has a hyphen, you need to wrap the key name in brackets and quotes. So...

`instructor["facial-hair"]`